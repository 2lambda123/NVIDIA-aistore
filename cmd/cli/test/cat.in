echo "0123456789" > $OBJECT_1.txt
ais bucket create ais://$BUCKET_1 | awk '{print $1 " " $2}'
ais object put $OBJECT_1.txt ais://$BUCKET_1/object_1.txt
rm -f $OBJECT_1.txt

ais object cat ais://$BUCKET_1/object_1.txt
