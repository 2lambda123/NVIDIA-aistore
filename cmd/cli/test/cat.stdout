"ais://$BUCKET_1" created
PUT "object_1.txt" to ais://$BUCKET_1
0123456789
